packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -229,4 +229,145 @@ void main() {
 
     expect(find.text('Subtitle'), findsOneWidget);
   });
+
+  testWidgets('ExpansionTile custom tween colors', (WidgetTester tester) async {
+    final Key expandedTitleKey = UniqueKey();
+    final Key collapsedTitleKey = UniqueKey();
+    final Key expandedIconKey = UniqueKey();
+    final Key collapsedIconKey = UniqueKey();
```

Comment:
I think the naming here is confusing because the tiles end up expanding and collapsing over time. Could we just use a single `ExpansionTile` and test the custom icon and title colors on that? In other words:
```dart
                ExpansionTile(
                  initiallyExpanded: true,
                  headerColorTween: headerColorTween,
                  iconColorTween: iconColorTween,
                  title: TestText('Expanded', key: titleKey),
                  children: const <Widget>[ListTile(title: Text('0'))],
                  trailing: TestIcon(key: iconKey),
                ),
```
just this single ExpansionTile and testing the custom colors when the tile is open and closed.